src/Resource/Otp.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+namespace SMSGlobal\Resource;
+
+use Cassandra\Date;
+use SMSGlobal\Exceptions\AuthenticationException;
+use SMSGlobal\Exceptions\InvalidPayloadException;
+use SMSGlobal\Exceptions\InvalidResponseException;
+use SMSGlobal\Exceptions\ResourceNotFoundException;
+
+/**
+ * Class Otp
+ * @package SMSGlobal\Resource
+ */
+class Otp extends Base
+{
+    /**
+     * @var string
+     */
+    protected $resourceUri = '/otp';
+
+    /**
+     * @param string $to
+     * @param string $text
+     * @param string|null $from
+     * @param int $codeExpirt
+     * @param int $length
+     * @param \DateTime|null $messageExpiryDateTime
+     *
+     * @return array
+     * @throws AuthenticationException
+     * @throws GuzzleException
+     * @throws InvalidPayloadException
+     * @throws InvalidResponseException
+     * @throws ResourceNotFoundException
+     */
+    public function send(string $to, string $text, string $from = null, string $codeExpiry = null, string $length = null, \DateTime $messageExpiryDateTime = null): array
```

Comment:
Looks like a lot of parameters to get wrong. How about using a `sendOptions` or `sendRequest` class for these?

Reply:
Existing send sms method is the same fashion so I dont want to introduce the new way in the minor version. May be next major release we change both.